component/topsql/topsql_test.go

```diff
@@ -94,27 +94,24 @@ func (s *testTopSQLSuite) TestInstances() {
 	var r []query.InstanceItem
 
 	r = nil
-	err = s.dq.Instances(int(now-41), int(now-40), &r)
+	err = s.dq.Instances(int(now-40), int(now-40), &r)
```

Comment:
It looks like the semantic of `start~end` have changed: Now start and end can be equal. Will this affect the web-front?

Reply:
Web-front won't depend on it.